<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractDialog</class>
 <widget class="QDialog" name="ExtractDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extract Files</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Box Root Folder</string>
       </property>
       <property name="buddy">
        <cstring>cmbRoot</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtName"/>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="btnRoot">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cmbRoot">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Import Sandbox Name</string>
       </property>
       <property name="buddy">
        <cstring>txtName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Import Sandbox from an archive</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="chkNoCrypt">
       <property name="text">
        <string>Import without encryption</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>cmbRoot</tabstop>
  <tabstop>btnRoot</tabstop>
  <tabstop>chkNoCrypt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
